modules/sc-mesh-secure-deployment/src/nats/comms_nats_controller.py

```diff
@@ -631,7 +664,7 @@ def __handle_received_config(
             f"HTTP Request Response: {response.text.strip()} {str(response.status_code).strip()}"
         )
 
-        data = json.loads(response.text)
+        # data = json.loads(response.text)
```

Comment:
Can this comment be removed? :call_me_hand:

Reply:
Yes.

Reply:
[061ee74](https://github.com/tiiuae/mesh_com/pull/373/commits/061ee74897362277bac34dd4cb1c5a6d80a9d2ff)